api/data/filters.sql

```diff
@@ -32,18 +32,18 @@ INSERT INTO public.page_filters VALUES ('effects-on-biodiversity', 'Positive;Neg
 INSERT INTO public.page_filters VALUES ('types-of-tools-or-platforms', 'Field Data Collection Apps;Precision Agriculture Technology;IoT Devices;Remote Sensing Platforms;Farm Management Software;Forest Management Software;Forest Inventory Tools;Traceability Systems;Research Databases');
 INSERT INTO public.page_filters VALUES ('data-storage', 'On-premises servers/local storage facilities;Cloud-based platforms;Data warehouses;Agricultural information management systems;Geographic Information Systems (GIS);Hybrid storage solutions (on-premises and cloud);secire data centres (advanced security measures)');
 INSERT INTO public.page_filters VALUES ('governance-model', 'Traditional/Subsistence Agriculture or Forestry;Cooperative Agriculture or Forestry;Corporate Agriculture or Forestry;Contract Farming or Forestry;Community Supported Agriculture or Forestry;Land trust and conservation Agriculture or Forestry;Other');
-INSERT INTO public.page_filters VALUES ('in-forestry', 'Reforestation ;Forest conservation - thinning, pruning, weed & pest control ;Felling ;Transportation of logs;Non-Timber Forest Products (NTFPs);Forest Fire Management;Forestry inventory and mapping;Other');
+INSERT INTO public.page_filters VALUES ('in-forestry', 'Reforestation;Forest conservation - thinning, pruning, weed & pest control;Felling;Transportation of logs;Non-Timber Forest Products (NTFPs);Forest Fire Management;Forestry inventory and mapping;Other');
 INSERT INTO public.page_filters VALUES ('age', 'Youth-centric (18-25);Middled-aged (25-50;Senior-aged (50+);Mixed-age');
 INSERT INTO public.page_filters VALUES ('education-level', 'High education;Advanced education;Technical and Vocational Education;Basic education');
 INSERT INTO public.page_filters VALUES ('experience-level', 'Experts;Mid-level professionals;Early-career/Entry level');
 INSERT INTO public.page_filters VALUES ('training-for-the-use-of-digital-technologies', 'Training programmes;Workshops;Seminars;Other');
 INSERT INTO public.page_filters VALUES ('percentage', '0-25%;26-50%;51-75%;75-100%');
 INSERT INTO public.page_filters VALUES ('type-of-device', 'Desktop computers;Laptop computers;Tablets;Smartphones;GPS devices;Agricultural machinery equipped with IoT (Internet of Things) sensors');
 INSERT INTO public.page_filters VALUES ('future-expansion/upgrade', 'Better connectivity/Infrastructure;More income/Access to funding;Standardisation efforts/Regulatory support;Better training and education');
-INSERT INTO public.page_filters VALUES ('yes/no/don''t-know', 'Yes;No;Don''t know');
-INSERT INTO public.page_filters VALUES ('answer', 'Facilitated;Completely;Hindered;Partially;No impact;Not at all;Advancements in existing technologies;<25%;Emergence of new technologies;<50%;No significant changes anticipated;<75%;>90%');
+-- INSERT INTO public.page_filters VALUES ('yes/no/don''t-know', 'Yes;No;Don''t know');
```

Comment:
if this a leftover? if not, we should remove it

Reply:
This was originally from the google sheet document but this is a little bit ambiguous and not a meaningful filter for the client.